packages/web/src/utils/canvas.ts

```diff
@@ -76,7 +79,7 @@ export function drawBezierCurve(
     ctx.bezierCurveTo(cp1.x, cp1.y, cp2.x, cp2.y, x, y);
     ctx.stroke();
     ctx.closePath();
-    drawPoint(ctx, cp1.x, cp1.y, 'red', 3, 0.75);
+    drawPoint(ctx, cp1.x, cp1.y, 'red', 'red', 1, 3, 0.75);
```

Comment:
Let's pull color from the arguments instead of hard coding it to `red`

Reply:
Yeah was meaning to ask why the color of the points are hardcoded as well, will fix both